ci/test/pipeline.template.yml

```diff
@@ -244,13 +244,13 @@ steps:
           composition: debezium-avro
           run: debezium-avro
 
-  - id: pg-cdc
-    label: "Postgres CDC test"
-    depends_on: build
-    plugins:
-      - ./ci/plugins/mzcompose:
-          composition: pg-cdc
-          run: pg-cdc
+  # - id: pg-cdc
```

Comment:
This makes me nervous! But I'm assuming that the tests are enabled in another PR that I will look at now 😄